static/index.html

```diff
@@ -84,6 +84,56 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. These also support other
             operating systems.</p>
+
+            <h2 id="early-stage-of-development">
+                <a href="#early-stage-of-development">Early stage of development</a>
+            </h2>
+            <p>GrapheneOS is a privacy / security research and engineering project that has been
+            under way for over 5 years. It recently became rebranded as GrapheneOS and is taking a
+            different direction based on obtaining funding for the research and development work
+            as a non-profit open source project rather than being a company. The reborn project is
+            still in a very early stage and lots of the past work on privacy and security has not
+            yet been restored for the new incarnation of the OS.</p>
+            <p>The grapheneos.org site is still being put together. It will have lots of
+            additional documentation and tutorials in the future along with coverage of various
+            software, firmware and hardware privacy/security topics.</p>
+            <p>GrapheneOS is being supported with funding and developers from various companies
+            and other organizations interested in contributing to this shared base for a feature
+            rich private and secure mobile operating system able to run many existing
+            applications. It will take more time to organize and deploy these resources in order
+            for the project to have a strong development team with proper infrastructure behind
+            it.</p>
+
+            <h3 id="overview">
```

Comment:
This should probably be integrated above into the existing overview. That's what I meant in terms of integrating it into the current page.

Reply:
Whoops! Sorry, one moment, will get it.